src/index.ts

```diff
@@ -814,6 +815,10 @@ export class ValidationService {
             }
 
             form.submit();
+
+            if (form.action != initialFormAction) {
+                form.action = initialFormAction;
+            }
```

Comment:
Straight assignment is probably sufficient.

Maybe `try/finally` just in case? No idea if it's even possible for `form.submit()` to throw.

```suggestion
            try {
                form.submit();
            } finally {
                form.action = initialFormAction;
            }
```